specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2023-03-01/PrometheusRuleGroups.json

```diff
@@ -0,0 +1,507 @@
+          },
+          "description": "the values for the alert rule resources."
+        }
+      },
+      "description": "Represents a collection of alert rule resources."
+    },
+    "PrometheusRuleGroupProperties": {
+      "description": "An Azure Prometheus rule group.",
+      "type": "object",
+      "required": [
+        "scopes",
+        "rules"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Rule group description."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable/disable rule group."
+        },
+        "clusterName": {
+          "type": "string",
+          "description": "Apply rule to data from a specific cluster."
+        },
+        "scopes": {
+          "$ref": "#/definitions/StringArray",
+          "description": "Target Azure Monitor workspaces resource ids. Only one scope currently supported."
+        },
+        "interval": {
+          "type": "string",
+          "description": "The interval in which to run the Prometheus rule group represented in ISO 8601 duration format. Should be between 1 and 15 minutes",
+          "format": "duration"
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrometheusRule"
+          },
+          "description": "Defines the rules in the Prometheus rule group.",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "PrometheusRule": {
+      "type": "object",
+      "description": "An Azure Prometheus alerting or recording rule.",
+      "required": [
+        "expression"
+      ],
+      "properties": {
+        "record": {
+          "description": "Recorded metrics name.",
+          "type": "string"
+        },
+        "alert": {
+          "description": "Alert rule name.",
+          "type": "string"
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable/disable rule."
+        },
+        "expression": {
+          "description": "The PromQL expression to evaluate. Every evaluation cycle this is evaluated at the current time, and the result recorded as a new set of time series with the metric name as given by 'record'.",
+          "type": "string"
+        },
+        "labels": {
+          "$ref": "#/definitions/StringDictionary",
+          "description": "Labels to add or overwrite before storing the result."
+        },
+        "severity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The severity of the alerts fired by the rule."
+        },
+        "for": {
+          "type": "string",
+          "description": "The amount of time alert must be active before firing.",
+          "format": "duration"
+        },
+        "annotations": {
+          "$ref": "#/definitions/StringDictionary",
+          "description": "Alert annotation key-value pairs."
```

Comment:
>"Alert annotation key-value pairs." [](http://example.com/codeflow?start=25&length=35)

>"Alert annotation key-value pairs." 
Not very familiar with Prometheus, but I didn't find this doc particularly enlightening.